pkg/knowledge_base2/graph_test.go

```diff
@@ -0,0 +1,74 @@
+package knowledgebase2
+
+import (
+	"testing"
+
+	construct "github.com/klothoplatform/klotho/pkg/construct2"
+	"github.com/klothoplatform/klotho/pkg/construct2/graphtest"
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/mock/gomock"
+)
+
+func TestDownstream(t *testing.T) {
```

Comment:
There is already a `TestDownstream` in `kbtesting/graph_test.go` (the different package was to prevent circular dependencies)

Reply:
we should try to keep testing local to the package otherwise its hard to know where it lives, but also there was clearly stuff missed from this tests so going to keep what i wrote

Reply:
I agree we should try, but since `properties` depends on `knowledgebase2` the test can't use those unless it's in a different package. I don't think we should keep the unit tests for a function split in two places. It shouldn't be hard to add a test for duplication to the one in `kbtesting`.

Reply:
we can mock properties if we need to, so i still think we should look to move it local to the package but for now i moved my tests into the kbtesting file

Reply:
Sure we can mock properties, but seems overkill compared to just using the ones in `properties`. Just seems like extra maintenance for the only benefit of which package/folder the test lives in.